fuzzware_pipeline/workers/tracegen.py

```diff
@@ -143,21 +149,27 @@ def gen_missing_maindir_traces(maindir, required_trace_prefixes, fuzzer_nums=Non
         if log_progress:
             logger.info(f"Generating traces took {time.time() - start_time:.02f} seconds for {len(input_paths)} input(s)")
     else:
+        m = mp.Manager()
+        processed_queue=m.Queue()
         num_processed = 0
+        job_args_multi = []
+        iteration = 0
         for input_path, bbl_trace_path, ram_trace_path, mmio_trace_path, bbl_set_path, mmio_set_path, bbl_hash_path in jobs_for_config:
-            gen_traces(str(config_path), str(input_path),
-                bbl_trace_path=bbl_trace_path, ram_trace_path=ram_trace_path, mmio_trace_path=mmio_trace_path,
-                bbl_set_path=bbl_set_path, mmio_set_path=mmio_set_path, bbl_hash_path=bbl_hash_path,
-                extra_args=extra_args, silent=not verbose
-            )
-            num_processed += 1
-
+            job_args_multi.append((str(config_path), str(input_path), bbl_trace_path, ram_trace_path, mmio_trace_path,
+                                   bbl_set_path, mmio_set_path, extra_args, not verbose, bbl_hash_path, processed_queue))
+            iteration += 1
+        p = mp.Process(target=multi_proc_manager, args=(gen_traces,job_args_multi))
```

Comment:
Same here. We want to be able to limit the number of cores/parallel processes used